apps/fabric-website/src/pages/Controls/AvatarPage/docs/android/AvatarOverview.md

```diff
@@ -1,5 +1,7 @@
 `AvatarView` displays either a photo or the initials of a person, entity, or group on top of a colored circular or square background.
 
-| Avatar style examples                                                                                                                                                                                            |                                                                                                                                                                                                                  |
-| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+<!-- prettier-ignore-start -->
+| Avatar style examples | |
+| - | - |
```

Comment:
I like this overall! Small thing, maybe use a few more dashes here to make it obvious that this is the line below the header?

Reply:
Sounds good! I was reading in a style guide to use 3+ dashes. Does 3 sound ok? I could see 5 working, too, but much more would get hard to keep track of consistently.

Reply:
Sure, that sounds good